lux/vislib/matplotlib/MatplotlibRenderer.py

```diff
@@ -81,6 +83,12 @@ def create_vis(self, vis, standalone=True):
             chart = LineChart(vis, fig, ax)
         elif vis.mark == "heatmap":
             chart = Heatmap(vis, fig, ax)
+        elif vis.mark == "geographical":
+            warnings.formatwarning = lux.warning_format
```

Comment:
Could we change the warning to be displayed via `df._message.add("...msg here...")`? The issue with the regular Python "red" warnings is that it can a bit jarring to users and given that this issue is somewhat minor, it makes more sense to go into the yellow button within our widget (which is only shown when we expand the message box).

You might notice that the df is not accessible in `MatplotlibRenderer:create_vis`, so it may be possible that we have to pass the message so that it is accessible to the main dataframe (so that we can edit `df._message`).